staramr/results/AMRDetectionSummary.py

```diff
@@ -76,3 +109,31 @@ def create_summary(self, include_negatives=False):
             df = df.merge(ds, on='Isolate ID', how='left').fillna(value={'Plasmid Genes': 'None'})
 
         return df.sort_index()
+
+    def create_detailed_summary(self, include_negatives=True):
+        df = self._resfinder_dataframe
+        df['Data Type']='Resistance'
+        ds = self._plasmidfinder_dataframe
+
+        column_names = ['Gene', 'Predicted Phenotype','%Identity', '%Overlap', 'HSP Length/Total Length','Contig', 'Start', 'End', 'Accession', 'Data Type']
+
+        ds = ds.reindex(columns=column_names)
+
+        if self._has_pointfinder:
+            dt = self._pointfinder_dataframe
+            dt['Data Type']='Resistance'
+            dt = dt.reindex(columns=column_names)
+            df = df.append(dt, sort=True)
+
+        if include_negatives:
+            df = self._include_detailed_negatives(df, ds)
+            df = df.fillna(" ")
```

Comment:
Could you fill with just an empty string `""` instead of a space?

Reply:
Yup, I will resolve this in my next commit, I only added a space to work around a column including plasmids with sensitive and realizing I didn't need that